Applications/Spire/Source/Ui/DropDownList.cpp

```diff
@@ -33,6 +34,7 @@ DropDownList::DropDownList(ListView& list_view, QWidget* parent)
   m_panel = new OverlayPanel(this, parent);
   m_panel->set_closed_on_blur(true);
   on_panel_style();
+  setFocusProxy(m_list_view);
```

Comment:
What's the reasoning for setting the focus proxy on line 37, where it splits up the work done on styling the panel?

Reply:
The purpose is not for styling. The OverlayPanel sets focus proxy to its body (DropDownList) and DropDownList sets focus proxy to the ListView. When setting focus on OverlayPanel, the ListView finally gets focus.